R/staggregate.R

```diff
@@ -0,0 +1,508 @@
+#' Aggregate Climate Data
+#'
+#' @param data The raster containing climate data
+#' @param variable The  climate variable of interest ('prcp, 'temp', and 'uv'
+#'   currently supported)
+#' @param daily_agg How to aggregate daily values ('sum' and 'average' currently
+#'   supported)
+#'   'polynomial', this is an integer indicating the degree.
```

Comment:
Should this top part include the `staggregate_*` parameters too? like degrees for polynomial and bins? I think we should have those documented as params but not sure the best way to do that or where those should go

Reply:
I'm not sure what to do there. I could duplicate that roxygen2 comment directly above each staggregate_* function, along with the transformation-specific parameters, giving each function it's own help page. Or I could use @describein which puts them all in the same help page. Let me know what you think.